### PR TITLE
Added example for GitLab URL configuration

### PR DESCRIPTION
**Pull request by felixkosmalla**
For first-time users it is not obvious that these settings need to go in the 'SOCIALACCOUNT_PROVIDERS'.

### CODE REVIEW

File: docs/providers.rst [hunk 1]
```diff
@@ -351,10 +351,12 @@ App registration (get your key and secret here)
 Development callback URL
     http://127.0.0.1:8000/accounts/github/login/callback/
 
-GitLab
+
+
```
Comment (pennersr) [inline docs/providers.rst]: Why drop the title?

File: docs/providers.rst [hunk 2]
```diff
@@ -351,10 +351,12 @@ App registration (get your key and secret here)
...
 ------
 
-The GitLab provider works by default with https://gitlab.com. It allows you
+The 
+provider works by default with https://gitlab.com. It allows you
```
Comment (pennersr) [inline docs/providers.rst]: This change does not seem to improve things ?

Comment (felixkosmalla) [inline docs/providers.rst]: also not intended

**Comment by pennersr** [comment]
Ok, I'll add the sample section later on...

**Comment by pennersr** [comment]
Closed via 81b990504d068a9a8840bfc2179e12d3e930df99

**Pull request closed without merging**